nav-app/src/app/techniques-search/techniques-search.component.html

```diff
@@ -1,34 +1,102 @@
+                                   (click)="toggleFieldEnabled(field.field)" [checked]="field.enabled">
+                            <label [for]="field.field" class="checkbox-custom-label noselect">{{field.label}}</label>
+                        </div>
+                    </div>
+                </mat-card-content>
+            </div>
+        </mat-card>
     </div>
-    <h1>properties searched</h1>
-    <div class="fields">
-        <div class="field" *ngFor="let field of fields">
-            <input [id]="field.field" class="checkbox-custom" type="checkbox" (click)="toggleFieldEnabled(field.field)" [checked]="field.enabled">
-            <label for="{{field.field}}" class="checkbox-custom-label noselect">{{field.label}}</label>
-        </div>
-    </div>
-    <div *ngIf="queryLength > 0">
-        <h1>results</h1>
-        <div class="allresults-buttons">
-            <button class="button" (click)="selectAll()">select all</button>
-            <button class="button" (click)="deselectAll()">deselect all</button>
-        </div>
-        <div class="results">
-            <table class="results" *ngIf="results.length > 0">
-                <tr *ngFor="let result of results">
-                    <td>
-                        <span *ngIf="result.isSubtechnique">{{result.parent.name}}: </span>{{result.name}}
-                    </td>
-                    <td><a href="{{result.url}}" target="_blank">view</a></td>
-                    <td><button class="button" (click)="select(result)">select</button></td>
-                    <td><button class="button" (click)="deselect(result)">deselect</button></td>
-                </tr>
-            </table>
-            <div class="no-results" *ngIf="results.length == 0"> no results </div>
-        </div>
+
+    <mat-divider></mat-divider>
+
+    <div class="sidebar-content">
+        <mat-accordion multi>
+        <mat-expansion-panel>
+            <mat-expansion-panel-header>
+                <mat-panel-title>
+                    <h4>Techniques ({{techniqueResults.length}})</h4>
+                </mat-panel-title>
+                <mat-panel-description>
+                </mat-panel-description>
+            </mat-expansion-panel-header>
+
+            <div>
+                <div class="allresults-buttons">
+                    <button class="button" (click)="selectAll(techniqueResults)">select all</button>
+                    <button class="button" (click)="deselectAll(techniqueResults)">deselect all</button>
+                </div>
+                <div class="results">
+                    <table class="results" *ngIf="techniqueResults.length > 0">
+                        <tr *ngFor="let result of techniqueResults">
+                            <td (mouseenter)="mouseEnter(result)" (mouseleave)="mouseLeave()">
+                                <span *ngIf="result.isSubtechnique">{{result.parent.name}}: </span>{{result.name}}
```

Comment:
Glad you kept the parent name as a prefix to the sub-technique name. I don't think I've mentioned it previously, but that's a standard we keep across the infrastructure -- if not shown hierarchically (not in the matrix or indented list, etc), the sub-technique name is always prepended with the parent name.